docs/UserGuide.md

```diff
@@ -220,7 +234,7 @@ Action | Format, Examples
 **Edit a member** | coming soon...
 **Create an event** | `createEvent n/EVENT_NAME [l/LOCATION] [d/DATE] [t/TAG]` <br> e.g.`createEvent n/Party l/Raffles Hall d/16-09-2023 t/Fun`
 **Delete an event** | `deleteEvent EVENT_INDEX` <br> e.g.`deleteEvent 1`
-**Edit an event** | coming soon...
+**Edit an event** | `editAttendance m/1 e/1 h/0 r/Absent due to Covid`
```

Comment:
should we display the format here also then show the example?

Reply:
I believe this should be placed under editing an Attendance so few lines down here.

Reply:
Agree with @wr1159